pages/osx/m.md

```diff
@@ -0,0 +1,23 @@
+# m
+
+> Swiss Army Knife for macOS.
+
+- Get the battery status:
+
+`m battery status`
+
+- Turn off the bluetooth:
+
+`m bluetooth off`
+
+- List available filesystems for formatting:
```

Comment:
```suggestion
- List available the filesystems for formatting:
```

Perhaps this reads better?

Reply:
No. I'm sure there is no need for "the" here.

Reply:
I agree, `the` should either be before the `available` (i.e. `List the available...`) or left as is (which I feel makes more sense). :+1:

Reply:
I would definitely leave as it is.